tox.ini

```diff
@@ -32,6 +32,8 @@ commands =
 extras =
     test
     docs
+# Must use editable package due to the way tests are discovered and run
+usedevelop = true
```

Comment:
We need to fix whatever causes the failure -- this change makes the `PURE_PYTHON` testing hard and / or not actually testing what it is supposed to do.

Reply:
I have removed the workaround so the error should show in the GHA test runs.

Reply:
While manually bisecting where the test failure happens (tests run fine without this change on `master`), I discoverd that running steps which make actual changes to the code (the currrent `lint` stuff) can break actual tests.  E.g., if I run `tox -e lint` against commit 94fd4cbf24fad0c98f4450bfea58b2e0365e470e, it introduces an actual `SyntaxError` into the code:

```bash
$ git checkout master
$ git pull
$ tox -e py38
...
  py38: OK (21.45=setup[15.50]+cmd[1.37,4.57] seconds)
  congratulations :) (21.74 seconds)
$ git checkout 94fd4cbf24fad0c98f4450bfea58b2e0365e470e
$ tox -e lint
...
$ tox - py38
...
======================================================================
ERROR: zope.interface (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: zope.interface
Traceback (most recent call last):
  File "/opt/Python-3.8.19/lib/python3.8/unittest/loader.py", line 470, in _find_test_path
    package = self._get_module_from_name(name)
  File "/opt/Python-3.8.19/lib/python3.8/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/home/tseaver/projects/Zope/Z3/zope.interface/.tox/py38/lib/python3.8/site-packages/zope/interface/__init__.py", line 53, in <module>
    from zope.interface.interface import Interface
  File "/home/tseaver/projects/Zope/Z3/zope.interface/.tox/py38/lib/python3.8/site-packages/zope/interface/interface.py", line 1172, in <module>
    from zope.interface.declarations import implementedBy
  File "/home/tseaver/projects/Zope/Z3/zope.interface/.tox/py38/lib/python3.8/site-packages/zope/interface/declarations.py", line 967
    isinstance(provides, Implements)  # noqa W503
    ^
SyntaxError: invalid syntax
```

Linting in ways that apply changes should be done manually, with the burden on the developer proposing those changes.  If a PR fails due to linting errors, there should be an easily documented step for the PR's author to perform to apply those fixes, and test them locally, before pushing to the PR.

Reply:
The branch config-with-c-code-template-acd39fc2 as it is locally on my machine does not apply any more changes when I run `bin/tox -elint`.

You are preaching to the choir, but I already explained that I was outvoted. I hate linting making changes myself.

Reply:
Can you point me to the issue?  The `--aggressive --aggressive` flag is particularly egregious.  Folks asking for such should be prepared to defend it (and do the work to fix what breaks).

Reply:
See https://github.com/zopefoundation/zope.interface/pull/322#issuecomment-2326706353 and the failed GHA run at https://github.com/zopefoundation/zope.interface/actions/runs/10684980548/job/29616689638. No idea why it only fails in that one manylinux run.

Reply:
The tests are now all green both locally and on GHA without `usedevelop`, so I'm closing this conversation.